checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -166,6 +168,14 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFbc) {
     }
 
     @Override
+    protected void addCheckedCodeDefaults(QualifierDefaults defs) {
+        // Exception parameters are always nonnull, so use that as the default
+        // rather than the CLIMB-to-the-top default.
```

Comment:
Exception parameters are not handled by CLIMB-to-top: there is no expression that would refine that type.
Why is this here instead of as a declarative annotation?